Mathlib/CategoryTheory/Limits/Shapes/Pullback/PullbackCone.lean

```diff
@@ -168,6 +166,12 @@ def ext {s t : PullbackCone f g} (i : s.pt ≅ t.pt) (w₁ : s.fst = i.hom ≫ t
   WalkingCospan.ext i w₁ w₂
 #align category_theory.limits.pullback_cone.ext CategoryTheory.Limits.PullbackCone.ext
 
+/-- The natural isomorphism between a pullback cone and the corresponding pullback cone
+reconstructed using `PullbackCone.mk`. -/
+@[simps!]
+def isoMkSelf (t : PullbackCone f g) : mk t.fst t.snd t.condition ≅ t :=
```

Comment:
This is usually called `eta`, and should probably go the other way. See docs#CategoryTheory.Limits.Cones.eta for comparison.

Similarly for pushouts.

Reply:
Oh thats good to know, thanks a lot!